cmd/rook/ceph/operator.go

```diff
@@ -43,8 +43,9 @@ func init() {
 	operatorCmd.Flags().DurationVar(&mon.HealthCheckInterval, "mon-healthcheck-interval", mon.HealthCheckInterval, "mon health check interval (duration)")
 	operatorCmd.Flags().DurationVar(&mon.MonOutTimeout, "mon-out-timeout", mon.MonOutTimeout, "mon out timeout (duration)")
 
-	operatorCmd.Flags().BoolVar(&csi.EnableRBD, "csi-enable-rbd", false, "whether enable ceph-csi rbd driver")
-	operatorCmd.Flags().BoolVar(&csi.EnableCephFS, "csi-enable-cephfs", false, "whether enable ceph-csi cephfs driver")
+	operatorCmd.Flags().BoolVar(&csi.EnableAll, "csi-enable", false, "enable all ceph csi support")
```

Comment:
If we're only exposing one flag in the operator env vars, do we need to expose three flags with more granularity here? Seems like we should have the same flag(s) in both places.

Reply:
I wasn't sure about that myself. I wanted to simplify it to one option but wanted to keep my changes small until I got some feedback. If you also prefer one option, I'll make a corresponding change.

Reply:
After the changes on the csi side to generate a single image, will we still have separate drivers for rbd and cephfs started? Or will there be a single driver? I think we should match what is being done there. If there are still two drivers, seems good to keep separate options for enabling cephfs and rbd, and no need for the new flag for enabling them both?

Reply:
Right now it would still be two "servers" but they'd share one image. The containers need to be started with different command lines.
That being the case I can remove the shared switch, I just thought it was convenient to be able to enable all ceph csi types at one. If you feel that's overkill I will drop it.

Reply:
I'd recommend keeping the two separate flags and no need for the enableAll flag.